File: server/order-service/src/main/java/com/sudo248/orderservice/controller/order/dto/PromotionDto.java
package com.sudo248.orderservice.controller.order.dto;

import com.sudo248.orderservice.repository.entity.order.Promotion;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class PromotionDto {
    private String promotionId = "";
    private String name = "";
    private Double value = 0.0;
}

File: client/mobile/Sudoo/app/src/main/java/com/sudo248/sudoo/data/dto/promotion/PromotionDto.kt
package com.sudo248.sudoo.data.dto.promotion

data class PromotionDto(
    val promotionId: String,
    val supplierId: String?,
    val value: Double,
    val name: String,
    val enable: Boolean,
    val image: String,
    val totalAmount: Int,
)
